src/OrchardCore.Modules/OrchardCore.Lucene/Handler/LuceneIndexingContentHandler.cs

```diff
@@ -82,7 +83,8 @@ private static async Task IndexingAsync(ShellScope scope, IEnumerable<ContentCon
 
                 foreach (var indexSettings in await luceneIndexSettingsService.GetSettingsAsync())
                 {
-                    var ignoreIndexedCulture = indexSettings.Culture == "any" ? false : context.ContentItem.Content?.LocalizationPart?.Culture != indexSettings.Culture;
+                    var contentItemCultureAspect = await contentManager.PopulateAspectAsync(context.ContentItem, new CultureAspect(){ Culture = null });
```

Comment:
Here we don't need to call GetContentCultureAsync() we already have the contentManager to use it directly.

Reply:
Agree and disagree ;) nothing wrong with using `GetContentCultureAsync()`  even we already have the contentManager to use it directly. Fix it once and use it everywhere, IMHO we don't wanna to enforce ourselves or everyone to write the changes in multiple places, which is the aim of the issue that you wrote. Please have a look to my comment [here](https://github.com/OrchardCMS/OrchardCore/pull/7307#issuecomment-711699574)